Add App to Teams/Add-TeamsAppInstallation.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$logfile = "C:\TeamsAppInstallationlog_$(get-date -format `"yyyyMMdd_hhmmsstt`").txt"
```

Comment:
Please add Synopsis block, i.e. from this template for inline help. https://github.com/TeamsAdminSamples/PowerShell/blob/main/z_Templates/New-PowerShell.ps1